tendrl/monitoring_integration/sync/dashbaord_sync.py

```diff
@@ -146,7 +146,7 @@ def update_dashboard(self, resource_type, resources, resource_json):
             alert_dashboard.check_duplicate(
                 resource_json, resources, resource_type
             )
-        if new_resources:
+        if new_resources and resource_json["dashboard"]["rows"]:
```

Comment:
to update dashbaord resource_json at least should have one row, because in updating i am duplicating the row and modifying based on new value